Doc/library/base64.rst

```diff
@@ -181,6 +184,26 @@ The modern interface provides:
    incorrectly padded or if there are non-alphabet characters present in the
    input.
 
+.. _base_64_base_85:
+
+Base85 Encodings
+-----------------
+
+Base85 encoding is not formally specified. Base85 encoding is a de facto standard,
+and different systems perform the encoding differently.
+
+The Ascii85 and Base85 functions in this module are two implementations of
```

Comment:
I'd do :func: / link to the functions rather than just saying name

Reply:
(Implemented)